pkg/e2e/operators/cloudingress/deletetapps2.go

```diff
@@ -0,0 +1,133 @@
+package cloudingress
+
+import (
+	"context"
+	"log"
+	"strings"
+
+	"time"
+
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	cloudingressv1alpha1 "github.com/openshift/cloud-ingress-operator/pkg/apis/cloudingress/v1alpha1"
+	"github.com/openshift/osde2e/pkg/common/constants"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	"github.com/openshift/osde2e/pkg/e2e/operators"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var _ = ginkgo.Describe(constants.SuiteInforming+TestPrefix, func() {
+	h := helper.New()
+	ginkgo.Context("Delete apps2 ingresscontroller", func() {
+		ginkgo.It("Should not be able to delete the ApplicationIngress that doesn't belong to CIO", func() {
```

Comment:
After reading through this test, I'm actually a bit confused about what is going on here? Could you organize it by "action" and add a comment explaining the "why" of each section? I.e. WHY are you removing annotations